superset-frontend/cypress-base/cypress/e2e/dashboard/drillby.test.ts

```diff
@@ -524,10 +524,141 @@ describe('Drill by modal', () => {
     });
 
     it('Bar Chart V2', () => {
-      testEchart('echarts_timeseries_bar', 'Bar Chart V2', [
+      const vizType = 'echarts_timeseries_bar';
+      const chartName = 'Bar Chart V2';
+      const drillClickCoordinates = [
         [70, 94],
-        [362, 68],
-      ]);
+        [362, 474],
```

Comment:
I'm not sure if it's due to Cypress upgrade or else but the previous coordinates were flaky. I have to manually tinker around to pick out the least "flaky" coordinates to pass local CI

Reply:
Yeah coordinates are always flakey in cypress as things can always add or subtract to it. You could try writing a function where you can pass in what you are looking for and then it'll generate an approximate window coordinate for you. I have that in our repo and it's still flakey but i'm not adding prs to make these updates as much